lib/data/multipleBackendGateway.js

```diff
@@ -48,23 +115,52 @@ const multipleBackendGateway = {
     },
 
     get: (objectGetInfo, range, reqUids, callback) => {
+        const key = objectGetInfo.key ? objectGetInfo.key : objectGetInfo;
```

Comment:
Okay. So what about if you're trying to retrieve an object that was stored before md-model-version 2 (or at least before the implementation of multiple backends) for the next line? `objectGetInfo.dataStoreName` would be undefined, right? Is the multiple backends feature backwards incompatible with objects stored before multiple backend implementation anyway? (Sorry, I just have a lot of questions because I'm not familiar with this feature)